docs/enforce-docs-up-to-date.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail -u
+
+current_docs=$(cat git_machete/docs.py)
+generated_docs=$(python docs/generate_docs.py dont_save)
+
+if [[ "$current_docs" != "$generated_docs" ]]; then
```

Comment:
What's the limit for env var contents (on Mac/on Linuxes/in Docker)? pls check if we're ever likely to hit that

Reply:
Alternatively, just use https://virtuslab.github.io/tips/#shell/output-comparing

Reply:
@PawelLipski Hmm, it looks like it's not that simple since it depends on how you set the env var value: https://devblogs.microsoft.com/oldnewthing/20100203-00/?p=15083 but per internet:

Linux: 128KiB -> https://unix.stackexchange.com/questions/336934/raise-128kib-limit-on-environment-variables-in-linux but some guy had env var with the size of 256 MB

MacOs: based on this post it is difficult to say and it also depends on the system resources > https://aplawrence.com/Unixart/variable_limits.html but the guy managed to get up to hundreds of MB

I couldn't find anything on docker limits, to sum up it's impossible to answer but it could be hundreds of MBs.

Reply:
Okay, looks safe generally then